src/main/java/com/stripe/param/CustomerCreateParams.java

```diff
@@ -280,12 +280,17 @@ public Builder setTaxInfo(TaxInfo taxInfo) {
     @SerializedName("custom_fields")
     Object customFields;
 
+    /** ID of the default payment method for the customer. */
+    @SerializedName("default_payment_method")
+    String defaultPaymentMethod;
```

Comment:
@remi-stripe 
flagging that `default_payment_method` is now on customer create as expected.